### PR TITLE
push all fixes for ticket #1184

### PR DESCRIPTION
**Pull request by georgeha**

### CODE REVIEW

**Comment by georgeha** [comment]
https://github.com/radical-cybertools/radical.pilot/issues/1184

File: src/radical/pilot/agent/lm/spark.py [hunk 1]
```diff
@@ -37,10 +37,10 @@ def lrms_config_hook(cls, name, cfg, lrms, logger):
         if not os.environ.get('SPARK_HOME'):
 	        logger.info("Downloading Apache Spark..")
 	        try:    
-		        VERSION = "1.5.2"
-		        subprocess.check_call("wget http://d3kbcqa49mib13.cloudfront.net/spark-1.5.2-bin-hadoop2.6.tgz".split())
-		        subprocess.check_call('tar -xzf spark-1.5.2-bin-hadoop2.6.tgz'.split())
-		        subprocess.check_call(("mv spark-1.5.2-bin-hadoop2.6 spark-" + VERSION).split())
+		        VERSION = "2.0.2"
+                        subprocess.check_call("wget http://d3kbcqa49mib13.cloudfront.net/spark-2.0.2-bin-hadoop2.7.tgz".split())
```
Comment (andre-merzky) [inline src/radical/pilot/agent/lm/spark.py]: This indentation looks off by one space?  but at least its not tabs anymore :D

Comment (georgeha) [inline src/radical/pilot/agent/lm/spark.py]: It works fine for me though!

Comment (andre-merzky) [inline src/radical/pilot/agent/lm/spark.py]: I am ssurprised by that, Python is usually picky about indentation changes like this.  Please do  fix it anyway, will you?  Thanks!

**Comment by andre-merzky** [comment]
thanks! :)

**Pull request merged**